### PR TITLE
Add support to short uuid on device.serviceVar.set

### PR DESCRIPTION
**Pull request by otaviojacobi**
Change-type: patch

Resolves: # 
HQ: <url> 
See: <url> 
Depends-on: <url> 
Change-type: major|minor|patch 

---
##### Contributor checklist

- [ ] Includes tests
- [ ] Includes typings
- [ ] Includes updated documentation
- [ ] Includes updated build output

### CODE REVIEW

File: lib/models/device.ts [hunk 1]
```diff
@@ -3392,18 +3392,24 @@ const getDeviceModel = function (
...
-				const deviceFilter = isId(uuidOrId)
-					? uuidOrId
-					: {
-							$any: {
-								$alias: 'd',
-								$expr: {
-									d: {
-										uuid: uuidOrId,
-									},
+				let deviceFilter;
```
Comment (thgreasi) [inline lib/models/device.ts:3395]: This atm is typed as `any`, so let's make it:
```suggestion
				let deviceFilter: ResourceObjFilter<Device>[keyof ResourceObjFilter<Device>];
```

File: lib/models/device.ts
Comment (thgreasi) [inline lib/models/device.ts:1]: Let's also copy paste the `.set` test case and make it use a short uuid
See: https://github.com/balena-io/balena-sdk/blob/6c3a9cc36e7f0c0b85ea3b373696651d1c9486c0/tests/integration/models/device.spec.js#L2381-L2389

**Review by thgreasi — COMMENTED** [review]
Other than that LGTM :+1:

File: lib/models/device.ts (hunk 1 above)
Comment (otaviojacobi) [reply lib/models/device.ts:3395]: nice catch

File: lib/models/device.ts
Comment (thgreasi) [reply lib/models/device.ts:1]: @otaviojacobi I would prefer to not monkey patch new props. Let's do the simple thing of adding a new test case especially for testing short uuids.

**Comment by otaviojacobi** [comment]
LGTM

**Pull request merged**